### PR TITLE
add rawkv-linearizability test

### PR DESCRIPTION
**Pull request by gengliqi**
Signed-off-by: Liqi Geng <gengliqiii@gmail.com>

### What problem does this PR solve? 

Add rawkv-linearizability test.

### CODE REVIEW

File: pkg/control/control.go [hunk 1]
```diff
@@ -57,7 +58,7 @@ func NewController(
 	cfg *Config,
 	clientCreator core.ClientCreator,
 	nemesisGenerators []core.NemesisGenerator,
-	clientRequestGenerator func(ctx context.Context, client core.Client, node clusterTypes.ClientNode, proc *int64, requestCount *int64, recorder *history.Recorder),
+	clientRequestGenerator func(ctx context.Context, clientId int, client core.Client, node clusterTypes.ClientNode, proc *int64, requestCount *int64, recorder *history.Recorder),
```
Comment (mahjonp) [inline pkg/control/control.go:61]: remove clientId here?

File: pkg/history/history.go [hunk 2]
```diff
@@ -65,7 +65,7 @@ func (r *Recorder) RecordResponse(proc int64, op interface{}) error {
 
 // RecordInvokeNemesis records nemesis invocation events on history file
 func (r *Recorder) RecordInvokeNemesis(nemesisRecord core.NemesisGeneratorRecord) error {
-	return r.record(-1, core.InvokeNemesis, nemesisRecord)
+	return r.record(1, core.InvokeNemesis, nemesisRecord)
```
Comment (mahjonp) [inline pkg/history/history.go:68]:
```suggestion
	return r.record(-1, core.InvokeNemesis, nemesisRecord)
```

**Comment by mahjonp** [comment]
Rest LGTM

File: go.mod [hunk 3]
```diff
@@ -38,7 +41,8 @@ require (
 
 replace google.golang.org/grpc => google.golang.org/grpc v1.26.0
 
-replace github.com/pingcap/pd => github.com/pingcap/pd v1.1.0-beta.0.20200106144140-f5a7aa985497
+// we use pingcap/pd and pingcap/pd/v4 at the same time, which will cause a panic because pd register prometheus metrics two times.
```
Comment (mahjonp) [inline go.mod:44]: I need wait client-go to update it's PD dependency.

File: cmd/rawkv-linearizability/main.go [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+	valueNum1MB     = flag.Int("ValueNum1MB", 200, "1MB value kind number")
+	valueNum5MB     = flag.Int("ValueNum5MB", 40, "5MB value kind number")
+)
+
+func main() {
+	flag.Parse()
+
+	var checkers []core.Checker
+	checkers = append(checkers, porcupine.Checker{})
+	// TODO should add more checker
```
Comment (mapleFU) [inline cmd/rawkv-linearizability/main.go:35]: Is here enough now?

File: cmd/util/suit.go [hunk 5]
```diff
@@ -151,15 +151,15 @@ func OnClientLoop(
...
 			log.Printf("%d %s: return %+v", procID, node, stringer.String())
 		} else {
 			log.Printf("%d %s: return %+v", procID, node, response)
 		}
-		isUnknown := true
-		if v, ok := response.(core.UnknownResponse); ok {
-			isUnknown = v.IsUnknown()
-		}
+
+		v := response.(core.UnknownResponse)
```
Comment (mapleFU) [inline cmd/util/suit.go:161]: Will us get other value other than `UnknownResponse`?

Comment (mapleFU) [reply cmd/util/suit.go:161]: Oh, seem that you modify it

File: cmd/rawkv-linearizability/main.go (hunk 4 above)
Comment (gengliqi) [reply cmd/rawkv-linearizability/main.go:35]: No...I will add one more checker at least

**Pull request merged**